doc/Customization-guide/How-to-contribute/Documentation-contribution-and-style-guide/index.md

```diff
@@ -221,33 +221,66 @@ Key differences between contributing source code and documentation are :
 
 ## Documentation preview
 
-* Install [Jekyll](https://jekyllrb.com/).
-See [Jekyll Installation page](https://jekyllrb.com/docs/installation/) for detail instructions.
-* Install [jekyll-sitemap](https://github.com/jekyll/jekyll-sitemap).
-Jekyll-sitemap can be installed using [RubyGems](https://rubygems.org/pages/download).
+1. Install required software :
+  * [Git](https://git-scm.com/)
+  * [Ruby](https://www.ruby-lang.org)
+  * [Jekyll](https://jekyllrb.com/)
+  * [Jekyll-sitemap](https://github.com/jekyll/jekyll-sitemap)
+  * [Jekyll-gist](https://github.com/jekyll/jekyll-gist)
+  * [Jekyll-paginate](https://github.com/jekyll/jekyll-paginate)
+  * [Rouge](https://github.com/jneen/rouge)
 
-  ```bash
-gem install jekyll-sitemap
-```
-* Install [rouge](https://github.com/jneen/rouge) for syntax highlighting.
-Rouge can be installed using [RubyGems](https://rubygems.org/pages/download).
+   Detailed installation instructions for selected platforms:
+<ul>
+<li style="list-style-type: none;">
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#Platform1">Ubuntu 14.04</a></li>
+  <li><a data-toggle="tab" href="#Platform2">Ubuntu 16.04</a></li>
+  <li><a data-toggle="tab" href="#Platform3">CentOS 7</a></li>
```

Comment:
Unfortunately, there are issues with different ruby versions (e.g. Ubuntu 14.04). And the only gemfile is ./nix/kaa-docs/Gemfile . For me, hidden directory means that it can disappear some day.